day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,18 @@
+def frog_poem(frogs)
+  while frogs > 1 do
+    puts "#{frogs} frogs sat on a log"
+    puts "eating some most delicious bugs."
+    puts "One jumped in the pool where its nice and cool,"
+    frogs -= 1
+    puts "then there were #{frogs} speckled frogs."
+  end
+  if frogs === 1
```

Comment:
Question for you to research: What's the difference in Ruby between `=`, `==`, and `===`?